packages/driver/src/cypress/script_utils.js

```diff
@@ -1,5 +1,5 @@
 const _ = require('lodash')
-const Promise = require('bluebird')
+const Bluebird = require('bluebird')
```

Comment:
I def prefer this - we should not call Bluebird "Promise", it's really confusing